mkdocs/docs/configuration.md

```diff
@@ -195,6 +198,86 @@ PyIceberg uses [S3FileSystem](https://arrow.apache.org/docs/python/generated/pya
 
 <!-- markdown-link-check-enable-->
 
+## Location Providers
+
+Iceberg works with the concept of a LocationProvider that determines file paths for a table's data. PyIceberg
+introduces a pluggable LocationProvider module; the LocationProvider used may be specified on a per-table basis via
+table properties. PyIceberg defaults to the [ObjectStoreLocationProvider](configuration.md#objectstorelocationprovider),
+which generates file paths that are optimized for object storage.
+
+### SimpleLocationProvider
+
+The SimpleLocationProvider places file names underneath a `data` directory in the table's storage location. For example,
```

Comment:
```suggestion
The `SimpleLocationProvider` places a table's data file names underneath a `data` directory in the table's storage location. For example,
```

Reply:
I think we should also call out that the "base location" is the `table.metadata.location` 

From the spec, https://iceberg.apache.org/spec/#table-metadata-fields

> The table's base location. This is used by writers to determine where to store data files, manifest files, and table metadata files.